proxy/js/styles.mjs

```diff
@@ -763,7 +763,13 @@ const sources = {
     url: `${origin}/electrification`,
     attribution,
     promoteId: 'id',
-  }
+  },
+  openhistoricalmap: {
+    type: 'vector',
+    tiles: [`https://vtiles.openhistoricalmap.org/maps/osm/{z}/{x}/{y}.pbf`],
+    attribution: 'OpenHistoricalMap', // TODO
```

Comment:
[This wiki page](https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/License) contains some advice on acknowledging OpenHistoricalMap. (I’m working on bringing it to the main site in OpenHistoricalMap/ohm-website#282 so it’s more official.)

Reply:
Yes, done!